edit/edit.js

```diff
@@ -671,14 +576,22 @@ window.onbeforeunload = () => {
     rememberWindowSize();
   }
   document.activeElement.blur();
-  if (isCleanGlobal()) {
+  if (isClean()) {
     return;
   }
   updateLintReportIfEnabled(null, 0);
   // neither confirm() nor custom messages work in modern browsers but just in case
   return t('styleChangesNotSaved');
 };
 
+function isClean() {
```

Comment:
I tried to avoid touching those global functions in edit.js.

Since this function is only used by `unbeforeunload` handler, maybe it shouldn't be put in the global?

Reply:
My point was that in case of trivial changes touching *might* be better than introducing a new function. Here I agree that it would make more sense to embed isClean in unbeforeunload.